docs/GCS-batchsink.md

```diff
@@ -44,6 +44,11 @@ If the format is a macro, only the pre-packaged formats can be used.
 The delimiter will be ignored if the format is anything other than 'delimited'.
 
 **Location:** The location where the gcs bucket will get created. This value is ignored if the bucket already exists.
+                           
+**Content Type:** The Content-Type entity is used to indicate the media type of the resource.
+
+**Custom Content Type:** The Custom Content Type is used when the value of Content-Type is set to other.
+User can provide specific Content-Type, different from the options in the dropdown.
```

Comment:
can add a link to list of other content types:
https://cloud.google.com/storage/docs/metadata